projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/wpcom-global-styles-view.scss

```diff
@@ -320,3 +359,5 @@
 		transform: none;
 	}
 }
```

Comment:
What's the deal with all the CSS being moved around? There doesn't seem to be a huge amount of actual changes apart from a specificity increase on some of the selectors, but the huge diff makes it very hard to check what actually changed.

Reply:
A lot of it has been refactored, both for the purposes of this PR and for readability and specificity. Happy to run through it, but if it looks fine on the frontend that's what's important. The CSS only belongs to this feature.